devops/settings.py

```diff
@@ -9,6 +9,12 @@
 
 LOG_LEVEL = "INFO"
 
+TEMPLATE_HEADER = """#
```

Comment:
If we change this to start with a clearer first line, e.g.

```yaml
# THIS FILE HAS BEEN AUTOMATICALLY GENERATED BY: poetry run invoke update-from-templates
# GENERATED FROM {file}, DO NOT MODIFY THIS FILE BY HAND
#
```

It will be enough to match the first line in the file to see if it was automatically generated by the tool and thus the cleanup of old templates will be easier.

Reply:
I actually both updated the header and also added logic to match the whole header, including the `{file}` placeholder. This should make it possible to use any header and properly detect it still.